pubspec.yaml

```diff
@@ -6,21 +6,21 @@ version: 1.1.0
 dependencies:
   flutter:
     sdk: flutter
-  meta: ^1.3.0
+  meta: ^1.7.0
   path_drawing: ^1.0.0
-  vector_math: ^2.1.0
+  vector_math: ^2.1.2
 
-  xml: ^5.0.0
+  xml: ^6.0.1
 
 dev_dependencies:
   flutter_test:
     sdk: flutter
-  path: ^1.8.0
-  test: ^1.16.0
+  path: ^1.8.1
+  test: ^1.21.1
 
 environment:
-  sdk: '>=2.12.0 <3.0.0'
-  flutter: ">=2.4.0-0.0.pre"
+  sdk: '>=2.17.0-0 <3.0.0'
+  flutter: ">=3.0.0"
```

Comment:
This seems to make CI unhappy. I think we can just leave it at the 2.4 with no upper bound. WDYT?

Reply:
Splash screen change was deprecated after 2.5 so I think it should be the lower bound.

Reply:
Upon review, the CI error appears to be in beta channel.
I propose we use `2.11.0-0.1.pre` as lower bound as it's the version that switches over to Dart 2.17.

<img src="https://user-images.githubusercontent.com/4012752/169405921-6a3dd24d-ea63-4eb9-a45e-b82356340e6a.png" width="500">